core/src/main/java/org/frankframework/http/rest/ApiListener.java

```diff
@@ -112,6 +113,8 @@ public enum HttpMethod {
 	private @Getter String headerParams = null;
 	private @Getter String contentDispositionHeaderSessionKey;
 	private @Getter String charset = null;
+	private @Getter @Nonnull Set<String> allowedParameterSet = Set.of();
+	private @Getter boolean allowAllParams = true;
```

Comment:
Ik ben niet helemaal blij met deze `allowAllParams` vlag maar ik zie nog niet een andere goede manier om oud gedrag toe te laten, maar ook de mogelijkheid te geven om helemaal geen request-parameters mee te nemen naar de sessie.

Reply:
Misschien dat je kan stellen;
> wanneer `allowAllParams` gevuld is moet hij strikt zijn en anders mag alles (oud gedrag).

Reply:
Maar voor hoe lang willen we dat toelaten, en wanneer willen we dat veranderen zodat je niet meer bij default "onveilig" bent?
Of denk je dat we dat voorlopig gewoon kunnen laten, met warning erbij?